doc/Learn/Maui/ThirdParty-MauiCommunityToolkit.md

```diff
@@ -9,22 +9,64 @@ The controls from MauiCommunityToolkit can be used in an Uno Platform applicatio
 1. Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
 
     ```dotnetcli
-    dotnet new unoapp -preset blank -maui -o MauiEmbeddingApp
+    dotnet new unoapp -preset blank -maui -o MauiCommunityToolkitApp
     ```
+### [Visual Studio](#tab/vs)
+> [!NOTE]
+> If you don't have the **Uno Platform Extension for Visual Studio** installed, follow [these instructions](xref:Uno.GetStarted.vs2022).
+
+- Launch **Visual Studio** and click on **Create new project** on the Start Window. Alternatively, if you're already in Visual Studio, click **New, Project** from the **File** menu.
+
+- Type `Uno Platform` in the search box
+
+- Click **Uno Platform App**, then **Next**
+
+- Name the project `MauiCommunityToolkitApp` and click **Create**
+
+At this point you'll enter the **Uno Platform Template Wizard**, giving you options to customize the generated application.
+
+- Select **Blank** in **Presets** selection
+
+- Select the **Platforms** tab and unselect **Desktop** and **Web Assembly** platforms
+
+- Select the **Features** tab and click on **.NET MAUI Embedding**
+
+ Click **Create** to complete the wizard
 
-1. Add a reference to the [CommunityToolkit.Maui NuGet package](https://www.nuget.org/packages/CommunityToolkit.Maui) to the MauiEmbeddingApp.MauiControls project.
+The template will create a solution with a single cross-platform project, named `MauiCommunityToolkitApp`, ready to run.
 
-1. In the `AppBuilderExtensions` class, on `MauiEmbeddingApp.MauiControls` project, update the `UseMauiControls` extension method to call the `UseMauiCommunityToolkit` method.
+For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
+
+### [Command Line](#tab/cli)
+
+> [!NOTE]
+> If you don't have the **Uno Platform dotnet new templates** installed, follow [dotnet new templates for Uno Platform](xref:Uno.GetStarted.dotnet-new).
+Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
+
+```bash
+dotnet new unoapp -preset blank -maui -platforms "android" -platforms "ios" -platforms "maccatalyst" -platforms "windows" -o MauiCommunityToolkitApp
+```
+
+This will create a new folder called **MauiCommunityToolkitApp** containing the new application.
+
+---
+
+1. Add a reference to the [MauiCommunityToolkitApp.Maui NuGet package](https://www.nuget.org/packages/CommunityToolkit.Maui) to the MauiCommunityToolkitApp.MauiControls project.
+
+1. In the `AppBuilderExtensions` class, on `MauiCommunityToolkitApp.MauiControls` project, update the `UseMauiControls` extension method to call the `UseMauiCommunityToolkit` method.
 
     ```cs
-    using CommunityToolkit.Maui;
+    using MauiCommunityToolkitApp.Maui;
```

Comment:
```suggestion
    using CommunityToolkitApp.Maui;
```

This should stay as `CommunityToolkit.Maui`, no? Since this is the reference to the actual [MAUI Community Toolkit NuGet Package](https://www.nuget.org/packages/CommunityToolkit.Maui)